ui_framework/src/components/form/form_checkbox/form_checkbox.js

```diff
@@ -0,0 +1,29 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+
+export const KuiFormCheckbox = ({ options, className, ...rest }) => {
+  const classes = classNames('kuiFormCheckbox', className);
+
+  return (
+    <div>
+      {options.map(function (option, index) {
```

Comment:
I have no idea how this should really be handled, right now I use a simple array, but that doesn't allow for things like passing `defaultChecked` on the actual input. Figured @cjcenizal could help here. I can say that checkboxes should always be in multiple sets, because switches are the single use checkbox variant.